### ISSUE TITLE
Server receiving requests multiple times

### ISSUE THREAD

**Issue by TheCommCraft** [TimMcCool/scratchattach#84]
Sometimes a server receives a request that was sent a long time ago again

**Comment by TimMcCool** [comment]
I think that can happen when the server loses connection to the websocket and reconnects, this causes the server to receive the current value of the TO_HOST cloud variable again . I’ll try to fix that bug

**Comment by TheCommCraft** [comment]
I found a fix. This should be the _update method of WsCloudEvents:
``` python
def _update(self):
    while True:
        try:
            data = self.connection.websocket.recv().split('\n')
            result = []
            for i in data:
                try:
                    result.append(json.loads(i))
                except Exception:
                    pass
            for activity in result:
                if "on_"+activity["method"] in self._events:
                    self._events["on_"+activity["method"]](self.Event(user=None, var=activity["name"][2:], name=activity["name"][2:], value=activity["value"], timestamp=time.time()*10000))
        except Exception:
            try:
                self.connection._connect(cloud_host=self.connection.cloud_host)
                self.connection._handshake()
                self.connection.websocket.recv()
            except Exception:
                if "on_disconnect" in self._events:
                    self._events["on_disconnect"]()
```

**Comment by TimMcCool** [comment]
> when connecting to Scratch’s cloud websocket, it sends the values of all cloud variables
This is how it should work and used to work, but for some reason Scratch does *not* send the values of all cloud variables when connecting to the websocket anymore.

I don't know what's causing the problem

**Comment by TimMcCool** [comment]
Sometimes Scratch's cloud websocket sends a message with the values of all cloud variables after connecting. The WsCloudEvents event handler considers this message as cloud activity, this causes the problem. Will be fixed soon

**Comment by TimMcCool** [comment]
Will be fixed in v1.2.6

**Comment by programORdie2** [comment]
Uhmm, i'm using v1.2.6 and i have still the same error.
